src/test/java/com/example/demo/app/membership/controller/MembershipControllerTest.java

```diff
@@ -0,0 +1,57 @@
+package com.example.demo.app.membership.controller;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import com.example.demo.app.enums.MembershipType;
+import com.example.demo.app.membership.dto.MembershipRequest;
+import com.google.gson.Gson;
+
+@ExtendWith(MockitoExtension.class)
+public class MembershipControllerTest {
+    
+    @InjectMocks
+    private MembershipController target;
+
+    private MockMvc mockMvc;
+    private Gson gson;
+
+    @BeforeEach
+    public void init() {
+        gson = new Gson();
+        mockMvc = MockMvcBuilders.standaloneSetup(target)
+                .build();
+    }
+
+    @Test
+    public void 멤버십등록실패_사용자식별값이헤더에없음() throws Exception {
+        // given
+        final String url = "/api/v1/memberships";
+
+        // when
+        final ResultActions resultActions = mockMvc.perform(
+                MockMvcRequestBuilders.post(url)
+                                      .content(gson.toJson(membershipRequest(10000, MembershipType.NAVER)))
+                                      .contentType(MediaType.APPLICATION_JSON));
+
+        // then
+        resultActions.andExpect(status().isBadRequest());
```

Comment:
`MockMvcResultMatchers.status` : Access to response status assertions


[Class MockMvcResultMatchers Method](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/result/MockMvcResultMatchers.html#status())